src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -0,0 +1,154 @@
+  }
+
+  renderBody() {
+    const {
+      group,
+      params: {orgId, tagKey},
+    } = this.props;
+    const {tag, tagValueList, tagValueListPageLinks} = this.state;
+    const sortedTagValueList: TagValue[] = sortBy(
+      tagValueList,
+      property('count')
+    ).reverse();
+
+    const issuesPath = `/organizations/${orgId}/issues/`;
+
+    const children = sortedTagValueList.map((tagValue, tagValueIdx) => {
+      const pct = tag.totalValues
+        ? `${percent(tagValue.count, tag.totalValues).toFixed(2)}%`
+        : '--';
+      const query = tagValue.query || `${tag.key}:"${tagValue.value}"`;
+      return (
+        <tr key={tagValueIdx}>
+          <td className="bar-cell">
+            <span className="label">{pct}</span>
+          </td>
+          <td>
+            <GlobalSelectionLink
+              to={{
+                pathname: issuesPath,
+                query: {query},
+              }}
+            >
+              {tag.key === 'user' ? (
+                <React.Fragment>
+                  <UserAvatar user={tagValue} size={20} className="avatar" />
+                  <span className="m-left">{this.getUserDisplayName(tagValue)}</span>
+                </React.Fragment>
+              ) : (
+                <DeviceName>{tagValue.name}</DeviceName>
+              )}
+            </GlobalSelectionLink>
+            {tagValue.email && (
+              <ExternalLink href={`mailto:${tagValue.email}`} className="external-icon">
+                <em className="icon-envelope" />
+              </ExternalLink>
+            )}
+            {isUrl(tagValue.value) && (
+              <a href={tagValue.value} className="external-icon">
+                <em className="icon-open" />
+              </a>
+            )}
+          </td>
+          <td>
+            <TimeSince date={tagValue.lastSeen} />
+          </td>
+        </tr>
+      );
+    });
+
+    return (
+      <TableWrapper>
+        <h3>
+          {tag.key === 'user' ? t('Affected Users') : tag.name}
+          <a
+            href={`/${orgId}/${group.project.slug}/issues/${group.id}/tags/${tagKey}/export/`}
+            className="btn btn-default btn-sm m-left"
+          >
+            {t('Export to CSV')}
+          </a>
+        </h3>
+        <table className="table table-striped">
+          <thead>
+            <tr>
+              <TableHeader width={20}>%</TableHeader>
+              <th />
+              <TableHeader width={300}>{t('Last Seen')}</TableHeader>
+            </tr>
+          </thead>
+          <tbody>{children}</tbody>
+        </table>
+        <Pagination pageLinks={tagValueListPageLinks} />
+        <p>
+          <small>
+            {t('Note: Percentage of issue is based on events seen in the last 7 days.')}
+          </small>
+        </p>
+      </TableWrapper>
+    );
+  }
+}
+
+const TableWrapper = styled('div')`
+  .m-left {
+    margin-left: ${space(1.5)};
+  }
+`;
```

Comment:
Also not sure about this approach, it's just because two components were both using `style={{marginLeft: 10}}`, and it felt like making two components was unnecessary but lmk!

also sidenote, this now sets the margin to 12px instead of 10px (since 10 isn't in spaces)

Reply:
Percy should get screenshots im assuming, but if not i can attach some

Reply:
@leeandher Percy only gets screenshots if there is an acceptance test of this view